scripts/get_all_linked_accounts_metadata.cdc

```diff
@@ -0,0 +1,61 @@
+import NonFungibleToken from "../contracts/utility/NonFungibleToken.cdc"
+import MetadataViews from "../contracts/utility/MetadataViews.cdc"
+import LinkedAccountMetadataViews from "../contracts/LinkedAccountMetadataViews.cdc"
+import LinkedAccounts from "../contracts/LinkedAccounts.cdc"
+
+pub struct LinkedAccountData {
+    pub let address: Address
+    pub let name: String
+    pub let description: String
+    pub let creationTimestamp: UFix64
+    pub let thumbnail: AnyStruct{MetadataViews.File}
+    pub let externalURL: MetadataViews.ExternalURL
+
+    init(
+        address: Address,
+        accountInfo: LinkedAccountMetadataViews.AccountInfo
+    ) {
+        self.address = address
+        self.name = accountInfo.name
+        self.description = accountInfo.description
+        self.creationTimestamp = accountInfo.creationTimestamp
+        self.thumbnail = accountInfo.thumbnail
+        self.externalURL = accountInfo.externalURL
+    }
+}
+
+/// Returns a mapping of metadata about linked accounts indexed on the account's Address
+///
+/// @param address: The main account to query against
+///
+/// @return A mapping of metadata about all the given account's linked accounts, indexed on each linked account's address
+///
+pub fun main(address: Address): {Address: LinkedAccountData} {
+    let linkedAccountData: {Address: LinkedAccountData} = {}
+
+    // Get reference to LinkedAccounts.Collection if it exists
+    if let collectionRef = getAccount(address).getCapability<
+            &LinkedAccounts.Collection{LinkedAccounts.CollectionPublic, MetadataViews.ResolverCollection}
+        >(
+            LinkedAccounts.CollectionPublicPath
+        ).borrow() {
```

Comment:
formatting is a little bit weird here. I don't know if that is intentional

Reply:
kind of, we end up calling that the angry cyclops in some conversations around the Cadence style guide, it's quite hard to decide where to break lines when specifying types in cadence

Reply:
Yeah trying to reduce the length of the top line here, but agree it looks a bit weird. Impartial to it either way